### PR TITLE
fix(docs): standardise id naming - part 2

### PR DESCRIPTION
**Pull request by shrutimantri**

### CODE REVIEW

File: core/src/main/java/io/kestra/plugin/core/kv/Set.java [hunk 1]
```diff
@@ -34,13 +34,18 @@
...
-              overwrite: true # whether to overwrite or fail if a value for that key already exists; default true
-              ttl: P30D # optional TTL"""
+                id: kv_store_set
+                namespace: company.team
+
+                tasks:
+                  - id: kv_set
+                    type: io.kestra.plugin.core.kv.Set
+                    key: my_variable
+                    value: "{{ outputs.query.uri }}"
```
Comment (wrussell1999) [inline core/src/main/java/io/kestra/plugin/core/kv/Set.java]: Output missing

File: core/src/main/java/io/kestra/plugin/core/flow/ForEach.java [hunk 2]
```diff
@@ -89,24 +90,24 @@
...
+                    values: 
+                      - value 1
+                      - value 2
+                      - value 3
+                      - value 4
+                    concurrencyLimit: 2
+                    tasks:
+                      - id: return
+                        type: io.kestra.plugin.core.debug.Return
+                        format: "{{ task.id }} with value '{{ taskrun.value }}'"
```
Comment (wrussell1999) [inline core/src/main/java/io/kestra/plugin/core/flow/ForEach.java]: Does `{{ taskrun.value }}` need to be wrapped in additional single quotes?

File: core/src/main/java/io/kestra/plugin/core/flow/ForEachItem.java [hunk 3]
```diff
@@ -168,7 +168,8 @@
                     wait: true
                     transmitFailed: true
                     inputs:
-                      json: '{{ json(read(taskrun.items)) }}'"""
+                      json: '{{ json(read(taskrun.items)) }}'
```
Comment (wrussell1999) [inline core/src/main/java/io/kestra/plugin/core/flow/ForEachItem.java]:
```suggestion
                      json: "{{ json(read(taskrun.items)) }}"
```

File: core/src/main/java/io/kestra/plugin/core/flow/EachSequential.java [hunk 4]
```diff
@@ -67,27 +67,28 @@
                         then:
                           - id: after_if
                             type: io.kestra.plugin.core.debug.Return
-                            format: 'After if {{ parent.taskrun.value }}'"""
+                            format: 'After if {{ parent.taskrun.value }}'
```
Comment (wrussell1999) [inline core/src/main/java/io/kestra/plugin/core/flow/EachSequential.java]:
```suggestion
                            format: "After if {{ parent.taskrun.value }}"
```

File: core/src/main/java/io/kestra/plugin/core/flow/ForEach.java [hunk 5]
```diff
@@ -79,7 +79,8 @@
                         then:
                           - id: after_if
                             type: io.kestra.plugin.core.debug.Return
-                            format: 'After if {{ parent.taskrun.value }}'"""
+                            format: 'After if {{ parent.taskrun.value }}'
```
Comment (wrussell1999) [inline core/src/main/java/io/kestra/plugin/core/flow/ForEach.java]:
```suggestion
                            format: "After if {{ parent.taskrun.value }}"
```

File: core/src/main/java/io/kestra/plugin/core/flow/If.java [hunk 6]
```diff
@@ -47,28 +47,28 @@
...
+                    required: true
+                
+                tasks:
+                  - id: if
+                    type: io.kestra.plugin.core.flow.If
+                    condition: "{{ inputs.string == 'Condition' }}"
+                    then:
+                      - id: when_true
+                        type: io.kestra.plugin.core.log.Log
+                        message: 'Condition was true'
```
Comment (wrussell1999) [inline core/src/main/java/io/kestra/plugin/core/flow/If.java]:
```suggestion
                        message: "Condition was true"
```

File: core/src/main/java/io/kestra/plugin/core/flow/If.java [hunk 7]
```diff
@@ -47,28 +47,28 @@
...
+                    else:
+                      - id: when_false
+                        type: io.kestra.plugin.core.log.Log
+                        message: 'Condition was false'
```
Comment (wrussell1999) [inline core/src/main/java/io/kestra/plugin/core/flow/If.java]:
```suggestion
                        message: "Condition was false"
```

File: core/src/main/java/io/kestra/plugin/core/flow/Template.java [hunk 8]
```diff
@@ -58,28 +58,30 @@
...
+                  - id: 1_return
+                    type: io.kestra.plugin.core.debug.Return
+                    format: "{{ task.id }} > {{ taskrun.startDate }}"
+                
+                  - id: 2_template
+                    type: io.kestra.plugin.core.flow.Template
+                    namespace: company.team
+                    templateId: template
+                    args:
+                      my-forward: "{{ inputs.with_string }}"
```
Comment (wrussell1999) [inline core/src/main/java/io/kestra/plugin/core/flow/Template.java]: Do we want it to be `my-forward` or `my_forward`?

File: core/src/main/java/io/kestra/plugin/core/flow/ForEach.java (hunk 2 above)
Comment (shrutimantri) [reply core/src/main/java/io/kestra/plugin/core/flow/ForEach.java]: Not required.. removing.

File: core/src/main/java/io/kestra/plugin/core/flow/Template.java (hunk 8 above)
Comment (shrutimantri) [reply core/src/main/java/io/kestra/plugin/core/flow/Template.java]: Its an argument and can depend on the template.

**Review by wrussell1999 — APPROVED** [review]

**Comment by wrussell1999** [comment]
This failed the CI but seems unrelated to the actual PR. Let me know if I should still merge this @Ben8t

**Comment by Ben8t** [comment]
Let double check with experts: @Skraye any idea why the CI would fails here ?
Want to be sure we don't break anything by merging this

**Comment by Skraye** [comment]
Retrying the CI make it pass, so its all good

**Pull request merged**